Makefile

```diff
@@ -162,15 +162,15 @@ format-ts-check:
 
 .PHONY: lint-go-ops
 lint-go-ops:
```

Comment:
Please use this opportunity to make these Makefile actions and CI workflows consistent across our projects (Solana, Cosmos, Starknet).